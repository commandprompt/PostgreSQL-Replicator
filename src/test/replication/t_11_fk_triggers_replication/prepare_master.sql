ALTER TABLE test1 ENABLE REPLICATION;
ALTER TABLE test2 ENABLE REPLICATION;
ALTER TABLE test3 ENABLE REPLICATION;
ALTER TABLE test4 ENABLE REPLICATION;
ALTER TABLE test5 ENABLE REPLICATION;
ALTER TABLE test1 ENABLE REPLICATION ON SLAVE 0;
ALTER TABLE test2 ENABLE REPLICATION ON SLAVE 0;
ALTER TABLE test3 ENABLE REPLICATION ON SLAVE 0;
ALTER TABLE test4 ENABLE REPLICATION ON SLAVE 0;
ALTER TABLE test5 ENABLE REPLICATION ON SLAVE 0;
ALTER TABLE test1 ENABLE REPLICATION ON SLAVE 1;
ALTER TABLE test2 ENABLE REPLICATION ON SLAVE 1;
ALTER TABLE test3 ENABLE REPLICATION ON SLAVE 1;
ALTER TABLE test4 ENABLE REPLICATION ON SLAVE 1;
ALTER TABLE test5 ENABLE REPLICATION ON SLAVE 1;
ALTER TABLE test1 ENABLE REPLICATION ON SLAVE 2;
ALTER TABLE test2 ENABLE REPLICATION ON SLAVE 2;
ALTER TABLE test3 ENABLE REPLICATION ON SLAVE 2;
ALTER TABLE test4 ENABLE REPLICATION ON SLAVE 2;
ALTER TABLE test5 ENABLE REPLICATION ON SLAVE 2;
