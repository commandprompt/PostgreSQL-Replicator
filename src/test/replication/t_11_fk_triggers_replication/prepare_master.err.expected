psql:.*:5: ERROR:  table "test5" does not have a primary key
psql:.*:10: ERROR:  table "test5" does not have a primary key
psql:.*:15: ERROR:  table "test5" does not have a primary key
psql:.*:20: ERROR:  table "test5" does not have a primary key
